pstoreds/ds_test.go

```diff
@@ -33,26 +33,24 @@ func TestDsPeerstore(t *testing.T) {
 
 func TestDsAddrBook(t *testing.T) {
 	for name, dsFactory := range dstores {
-		t.Run(name, func(t *testing.T) {
-			t.Run("Cacheful", func(t *testing.T) {
-				t.Parallel()
+		t.Run(name+" Cacheful", func(t *testing.T) {
```

Comment:
Can we keep the hierarchical `[datastore]/{Cacheful,Cacheless}/[testcase]` format in the full test name, or is the slash a reserved char?

Reply:
Sorry, didn't see this review. I will be adding some more benchmarks soon, so I will add that in then.